### PR TITLE
Upgrade Ballerina version to 2201.5.0

### PR DESCRIPTION
**Pull request by Thiyan**

### CODE REVIEW

File: r4/athena/Ballerina.toml [hunk 1]
```diff
@@ -16,4 +16,9 @@ observabilityIncluded = true
 [[dependency]]
 org = "ballerinax"
 name = "health.clients.fhir"
-version="1.0.1"
+version="1.0.2"
+
+[[dependency]]
+org = "ballerinax"
+name = "health.base"
+version="1.0.2"
```
Comment (AnjanaSenanayake) [inline r4/athena/Ballerina.toml]: Not reformatted

File: r4/cerner/Ballerina.toml [hunk 2]
```diff
@@ -16,4 +16,9 @@ observabilityIncluded = true
 [[dependency]]
 org = "ballerinax"
 name = "health.clients.fhir"
-version="1.0.1"
+version="1.0.2"
+
+[[dependency]]
+org = "ballerinax"
+name = "health.base"
+version="1.0.2"
```
Comment (AnjanaSenanayake) [inline r4/cerner/Ballerina.toml]: Needs reformatting

File: r4/epic/Ballerina.toml [hunk 3]
```diff
@@ -16,9 +16,9 @@ observabilityIncluded = true
...
 org = "ballerinax"
 name = "health.clients.fhir"
-version="1.0.1"
+version="1.0.2"
 
 [[dependency]]
 org = "ballerinax"
 name = "health.base"
-version="1.0.1"
+version="1.0.2"
```
Comment (AnjanaSenanayake) [inline r4/epic/Ballerina.toml]: Reformattings

File: r4/epic/Ballerina.toml [hunk 4]
```diff
@@ -3,10 +3,10 @@ org = "ballerinax"
 name = "health.fhir.templates.r4.epicconnect"
 displayName = "Epic FHIR API"
 description = "Exposes Epic FHIR APIs as managed APIs"
-version = "1.0.2-snapshot"
+version = "1.0.2"
 authors = ["Ballerina"]
 keywords = ["Healthcare", "FHIR", "Epic"]
-distribution = "2201.4.1"
+distribution = "2201.5.0"
 template=true
```
Comment (AnjanaSenanayake) [inline r4/epic/Ballerina.toml]: Reformattings

File: r4/athena/Ballerina.toml (hunk 1 above)
Comment (Thiyan) [reply r4/athena/Ballerina.toml]: Updated

File: r4/cerner/Ballerina.toml (hunk 2 above)
Comment (Thiyan) [reply r4/cerner/Ballerina.toml]: Updated

File: r4/epic/Ballerina.toml (hunk 3 above)
Comment (Thiyan) [reply r4/epic/Ballerina.toml]: Updated

File: r4/epic/Ballerina.toml (hunk 4 above)
Comment (Thiyan) [reply r4/epic/Ballerina.toml]: Updated

**Review by AnjanaSenanayake — APPROVED** [review]

**Review by ThishaniLucas — APPROVED** [review]

**Pull request merged**